### PR TITLE
dev-python/rebulk: verbump to 0.7.3

### PR DESCRIPTION
**Pull request by Coacher**
I've also bumped the required rebulk version in guessit-9999 as did upstream.
Now that guessit:master was fixed, I've added python-3.5 to it.
Build and test log: https://bpaste.net/show/50d03a366c1f

CC @gentoo/proxy-maint

### CODE REVIEW

**Comment by kensington** [comment]
@gentoo/proxy-maint

File: dev-python/rebulk/rebulk-0.7.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library that performs advanced searches in strings"
+HOMEPAGE="https://github.com/Toilal/rebulk https://pypi.python.org/pypi/rebulk"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
```
Comment (gktrk) [inline dev-python/rebulk/rebulk-0.7.3.ebuild]: rebulk/toposort.py is licensed under Apache

Comment (Coacher) [inline dev-python/rebulk/rebulk-0.7.3.ebuild]: Fixed.

**Comment by Coacher** [comment]
My paste has expired. Please ping me if you need a new one to be uploaded.

**Comment by Coacher** [comment]
Ping

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/3c76d25...2bafef7. Thanks!

**Comment by Coacher** [comment]
Thanks for the merge, @gktrk.

**Pull request closed without merging**